polaris-tokens/scripts/toValues.ts

```diff
@@ -0,0 +1,44 @@
+import fs from 'fs';
+import path from 'path';
+
+import {metaThemeVariants, metaThemeDefault} from '../src/themes';
+import {extractMetaThemeValues} from '../src/themes/utils';
+
+const outputDir = path.join(__dirname, '../build');
+
+export async function toValues() {
+  await fs.promises.mkdir(outputDir).catch((error) => {
+    if (error.code !== 'EEXIST') {
+      throw error;
+    }
+  });
+
+  const themeDefault = extractMetaThemeValues(metaThemeDefault);
+
+  await fs.promises.writeFile(
+    path.join(outputDir, 'index.ts'),
+
+    [
+      `export * from '../src/index';`,
+      Object.entries(themeDefault).map(createExport),
+      createExport(['tokens', themeDefault]),
```

Comment:
Note: The exports will be removed in the v12 release as they will no longer be relevant in the new theming architecture.